samples/app-cache-meetings/nodejs/assets/sample.json

```diff
@@ -3,13 +3,13 @@
     "name": "officedev-microsoft-teams-samples-app-caching-in-meetings-nodejs",
     "source": "officeDev",
     "title": "App Caching in Meetings",
```

Comment:
Is this correct?
"title": "App Caching in Meetings",

You also have personal tab scope.

Reply:
comment fixed